scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/struct/ClassParameterMemberParser.java

```diff
@@ -23,25 +26,50 @@
 
 		final ArrayList<SynthesizedParameterMember<?>> items = new ArrayList<>();
 		final ArrayList<ValidityProblem> problems = new ArrayList<>();
-		final Set<String> names = new HashSet<>();
 
 		// NB: Reject abstract classes.
 		org.scijava.struct.Structs.checkModifiers(source.getName() + ": ", problems, source.getModifiers(), true, Modifier.ABSTRACT);

```

Comment:
Why is this not imported? Is there more than one `Structs` class in different packages?

Reply:
Unfortunately, there is [another](https://github.com/scijava/incubator/blob/8784476c756492135b1c7d1dd4578428e0867924/scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/util/Inject.java#L24). Fixed in 8784476c

Reply:
That other `Structs` class was deleted in 4a031a88. 👋